ipatests/test_integration/test_backup_and_restore.py

```diff
@@ -332,6 +332,8 @@ def _full_backup_restore_with_DNS_zone(self, reinstall=False):
                                      '--uninstall',
                                      '-U'])
 
+            version = tasks.get_package_version(
+                self.master, '*ipa-server-dns')
```

Comment:
If we want to install the same version I think we need to include %{RELEASE} as well as %{VERSION} to be sure they are literally the same.